capa/v29.6.0/release.yaml

```diff
@@ -0,0 +1,136 @@
+# Generated with:
+# devctl release create --provider aws --name 29.6.0 --base 29.5.0 --component cluster-aws@2.5.0 --app aws-nth-bundle@1.2.0 TODO RE-RUN WITH REAL VERSIONS
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: aws-29.6.0
+spec:
+  apps:
+  - dependsOn:
+    - cloud-provider-aws
+    name: aws-ebs-csi-driver
+    version: 2.30.1
+  - dependsOn:
+    - prometheus-operator-crd
+    name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+  - name: aws-nth-bundle
+    version: 1.2.0-2da626ef1c0001aa4f1f38d4c9732f16bedd8a6b
+    catalog: default-test
```

Comment:
The test catalog doesn't seem to be used. I fixed it manually in E2E clusters.

Reply:
According to the code it should be used: https://github.com/giantswarm/cluster-aws/blob/v2.5.0/helm/cluster-aws/templates/aws-nth-app.yaml#L68. This is retrieving the catalog from the release CR.